### PR TITLE
Issue 10

### LINKED ISSUE #10: New op: fitted
screen shot 2015-11-18 at 2 19 39 pm

### PR DESCRIPTION
**Pull request by xiam**
Adding "fitted" op and "canvas" parameter. The JPEG canvas looks ugly because JPEG does not have transparency. Maybe we could add another issue to add a `canvasbg` parameter for that case?

Closes #10.

### CODE REVIEW

**Comment by pkieltyka** [comment]
can you try this PR with &format=png  .. and use a png as the source image.. this should give you a proper transparency

**Comment by xiam** [comment]
@pkieltyka I tried with JPEG, PNG and animated GIFs as sources. The generated images should be written to testdata/

File: imagick/imagick.go [hunk 1]
```diff
@@ -251,9 +254,15 @@ func (i *Image) sizeFrames(sz *imgry.Sizing) error {
 		i.mw = i.mw.CoalesceImages()
 	}
 
+	if sz.Canvas != nil {
+		// If the user requested a canvas.
+		canvas = imagick.NewMagickWand()
+		bg = imagick.NewPixelWand()
+		bg.SetColor("transparent")
```
Comment (pkieltyka) [inline imagick/imagick.go]: Can you make sure that you are destroying/releasing both the canvas (MagickWand) and bg (PixelWand) once you're done with it...? imagick is notorious for leaking memory

File: imagick/imagick.go [hunk 2]
```diff
@@ -262,6 +271,7 @@ func (i *Image) sizeFrames(sz *imgry.Sizing) error {
 		if err != nil {
 			return err
```
Comment (pkieltyka) [inline imagick/imagick.go]: ie. if this hits.. we return an error and never release the canvas/bg

**Pull request merged**